README.md

```diff
@@ -142,16 +140,77 @@ If you need to manually apply migrations:
     * Create credentials without role
     * Press "Create key" and select JSON format
     * JSON file will be downloaded automatically
-3. Getting Google client ID and Google client secret
```

Comment:
I have a feeling that 'Setting up Google account' section should be moved entirely.

So usages of drive_service_key.json may be marked optional in context of auth setup.

Reply:
Drive service is still required for reports

Reply:
Oh, sorry, messed it up.

Reply:
Then step for Google account setup should only stay in context of drive_service_key.json file.
And new subsections in deploy sections should be introduced to link to auth setup.

Reply:
Consider that:
* You need to set up google account anyways, whether you need authentication with it or now and
* One section of `Set up an environment` step depend on google account setup

Wouldn't it make more sense for step `Set up Google account` (currently step 2) to come before `Set up an environment` (currently step one)? I.e. Step 1 would be `Set up Google account` and step 2 would be `Set up an environment`.

Even in current documentation a section of `Set up an environment` forward-references google account setting up, so regardless of this PR I think it would be a good change

Reply:
What is deploy section? Is it `Set up an environment`?

Reply:
Regarding moving `Set up Google account` to the first step - totally agree.

Deploy sections: `Deployment script` and `Manual build and run` (at least `Run application` subsection). There are no notions of auth setup.

Reply:
Got it. Let me add auth setup to those sections and restructure this a bit

Reply:
All good, just two things:
- `Deployment script` point 2 `Set up an environment:` got 1.* numbered subpoints
- `Manual build and run` -> `Run application` indentation is broken.

Reply:
What is wrong with numbered subpoints? This section already has numbered subpoints in the master. Am I missing something?

As for the indentation in `Run application`, what indentation should it have to be considered not broken? To me it looks broken in master as it is not consistent:
* Line #37 contains 8 spaces
* Lines #38, #43, #47 #51 contain 4 spaces

But what's more important, all these lines are empty, so it does not make much sense to indent them at all. As far as I know, it's a pretty standard practice to keep empty lines empty, and the is a good reason for that:
https://stackoverflow.com/questions/5917956/why-is-indentation-in-empty-lines-bad

Therefore I removed the indentation for these section to be both consistent (no 8 vs 4 spaces) and what seems to me better